doc/introduction/operations.rst

```diff
@@ -42,25 +42,45 @@ Operator functions
 ------------------
 
 Various functions and transforms are available for manipulating operators,
-and extracting information.
+and extracting information. These can be broken down into two main categories:
+
+Operator transform functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. autosummary::
 
     ~pennylane.adjoint
     ~pennylane.ctrl
     ~pennylane.cond
+    ~pennylane.op_sum
+    ~pennylane.prod
+    ~pennylane.s_prod
+
+These operator functions act on operators to produce new operators.
+
+>>> op = qml.prod(qml.PauliX(0), qml.PauliZ(1))
+>>> op = qml.op_sum(qml.Hadamard(0), op)
+>>> op = qml.s_prod(1.2, op)
+1.2*(Hadamard(wires=[0]) + PauliX(wires=[0]) @ PauliZ(wires=[1]))
+
+Operator information functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autosummary::
+
     ~pennylane.matrix
     ~pennylane.eigvals
     ~pennylane.generator
```

Comment:
Note that generator should return an operator as well

Reply:
I guess the main distinction is 'functions that return operators' vs. 'functions that don't return operators (could be arrays, strings, etc.)'

Reply:
Would it be sufficient to just move generator up? Or do you think the title / description should change. Personally, the description might be a bit off as I don't really think of transforming an op to its generator as much as asking for the generator of the op

Reply:
Generator moved up 👍🏼